version.txt

```diff
@@ -1 +1 @@
-2.0.10
```

Comment:
I am not sure if a major version upgrade is warranted - it might even prevent automatic updates for some people who allow all `2.0.x` versions to be installed.

Reply:
The problem is the `audience` feature that was merged before. Ideally we would need to cherry pick just this fix and nothing more into 2.0.11, but not sure how we can do it. Revert the merge maybe?

Reply:
Ah, interesting. I missed that. Since it is a new feature, you can get by with just increasing the minor version, i.e. `2.1.0`. Or is it a breaking change?

Reply:
done